src/main/resources/liquibase/workbench_changelog/20_1_0.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+				   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+				   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+	<changeSet author="nahuel" id="v20.1.0-1">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="0">
+				SELECT count(*) FROM workbench_tool WHERE name = 'program_settings_manager'
+			</sqlCheck>
+		</preConditions>
+		<comment>
+			Add Manage Program Settings Beta sidebar item.
+		</comment>
+		<sql dbms="mysql" splitStatements="true">
+			INSERT INTO workbench_tool (`name`, `group_name`, `title`, `version`, `tool_type`, `path`, `parameter`, `user_tool`)
+			VALUES ('program_settings_manager', 'program_settings_manager', 'Manage Program Settings Beta', '1.0', 'WEB', '/ibpworkbench/controller/jhipster#program-settings-manager', '', 0);
+
+			INSERT INTO workbench_sidebar_category_link (`tool_name`, `sidebar_category_id`, `sidebar_link_name`, `sidebar_link_title`, `rank`)
+			VALUES ('program_settings_manager', 7, 'program_settings_manager', 'Manage Program Settings Beta', 2);
+		</sql>
+	</changeSet>
+
+	<changeSet author="nahuel" id="v20.1.0-2">
```

Comment:
@nahuel-soldevilla this is only for testing right? We will need to add another changeset to delete this permission when we disable the old module

Reply:
posting here what we discussed:

only for testing, yes.
About **selectable**, yes it's possible but we can also disregard any intermediate state when we do the final migration, and copy the exact same state as the old permission when we disable it (the roles that have the old permission are automatically given the new one)
This also give us more opportunities to test it exactly as it's going to be in the end, as a selectable permission.
Any intermediate association with the the beta permission will be disregarded later.

The **Api**/**PreAuthorize** can be tested by assigning the final permission name (MANAGE_PROGRAM_SETTINGS instead of MANAGE_PROGRAM_SETTINGS_BETA) and giving both permission through the side admin, one for accessing the module in the side menu, and the other for giving access to the api calls. This way in the final migration we don't need to rename permission in the the preauthorize.

Reply:
I think we can simplify the changeset if we are going to disable the old module in 20.1 
If testing passes and before merging we can discuss with Mariano and decide how to proceed
@nahuel-soldevilla